webserver/webserver/src/main/java/io/helidon/webserver/ServerListener.java

```diff
@@ -346,7 +346,8 @@ private void listen() {
                                                     socket,
                                                     serverChannelId,
                                                     router,
-                                                    tls);
+                                                    tls,
```

Comment:
ListenerConfig is available through `listenerContext`, not sue we need to send it separately, probably not really an optimization

Reply:
Fixed